x-pack/plugins/security_solution/public/management/components/management_page_view.tsx

```diff
@@ -4,15 +4,36 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React, { memo, useMemo } from 'react';
+import React, { memo, useCallback, useMemo } from 'react';
 import { i18n } from '@kbn/i18n';
-import { useParams } from 'react-router-dom';
+import { useParams, useHistory } from 'react-router-dom';
 import { PageView, PageViewProps } from '../../common/components/endpoint/page_view';
 import { ManagementSubTab } from '../types';
-import { getManagementUrl } from '..';
+import { SecurityPageName } from '../../app/types';
+import { useFormatUrl } from '../../common/components/link_to';
+import { getEndpointListPath, getPoliciesPath } from '../common/routing';
 
 export const ManagementPageView = memo<Omit<PageViewProps, 'tabs'>>((options) => {
+  const history = useHistory();
+  const { formatUrl, search } = useFormatUrl(SecurityPageName.management);
   const { tabName } = useParams<{ tabName: ManagementSubTab }>();
+
+  const goToEndpoint = useCallback(
```

Comment:
We should use `useNavigateByRouterEventHandler()` for these since that hook correctly handles keyboard interactions/expectations